Cider Remote/Views/MusicPlayerView.swift

```diff
@@ -937,9 +942,86 @@ class MusicPlayerViewModel: ObservableObject {
             }
         }
         self.liveActivity.device = device
+        
+        setupAudioSession()
+        setupSilentAudioPlayer()
+        setupRemoteCommandCenter()
+        updateNowPlayingInfo()
+    }
+    
+    private func setupRemoteCommandCenter() {
```

Comment:
These need to be handled differently somehow, they fire twice or don't fire at all.